databricks/koalas/missing/indexes.py

```diff
@@ -119,7 +119,7 @@ class _MissingPandasLikeMultiIndex(object):
 
     # Properties
     is_all_dates = unsupported_property('is_all_dates')
-    levshape = unsupported_property('levshape')
+    shape = unsupported_property('shape')
```

Comment:
why do you add this?? i thnik it is already implemented ?

```python
>>> index = Index([1, 2, 3])
>>> index.shape
(3,)
```

not pretty sure, but i guess maybe that is the reason of build failure  :)

Reply:
in the above question, "method for checking the python version" means like that?

```python
>>> import sys
>>> sys.version
'3.7.2 (v3.7.2:9a3ffc0492, Dec 24 2018, 02:44:43) \n[Clang 6.0 (clang-600.0.57)]'
```